packages/abi-gen-templates/partials/callAsync.handlebars

```diff
@@ -1,3 +1,10 @@
+/**
+ * Calls the method
```

Comment:
`Sends a read-only call to the contract method returning the result that would happen were one to submit an Ethereum transaction to this method given the current state of the blockchain. Calls do not cost gas since they don't modify state.`